domain/robstoll-lib/atrium-domain-robstoll-lib-jvm/src/main/kotlin/ch/tutteli/atrium/domain/robstoll/lib/creating/pathAssertions.kt

```diff
@@ -16,3 +18,6 @@ fun <T : Path> _exists(assertionContainer: Expect<T>): Assertion =
 
 fun <T : Path> _existsNot(assertionContainer: Expect<T>): Assertion =
     ExpectImpl.builder.createDescriptive(assertionContainer, NOT_TO, RawString.create(EXIST)) { it.notExists }
+
+fun <T : Path> _parent(assertionContainer: Expect<T>): ExtractedFeaturePostStep<T, Path> =
+    ExpectImpl.feature.manualFeature(assertionContainer, PARENT) { parent }
```

Comment:
calling parent is actually not safe this way. It could be `null` in which case kotlin would turn it into a NullPointerException. Extract the feature with ` ExpectImpl.feature.extractor(assertionContainer)`

Reply:
done